src/helpers/data_normalization.ts

```diff
@@ -45,3 +59,29 @@ export function* iterateItemIdsPositions(sheetId: UID, itemIdsByZones: Record<st
     }
   }
 }
+
+function getCanonicalRepresentation(item: any): string {
```

Comment:
Maybe we can use number as the canonical representation of number and have number | string as the reverseLookup map key :)